code/game/objects/items/devices/scanners.dm

```diff
@@ -401,7 +401,23 @@ GENE SCANNER
 			if(ishuman(C))
 				var/mob/living/carbon/human/H = C
 				if(H.is_bleeding())
-					render_list += "<span class='alert ml-1'><b>Subject is bleeding!</b></span>\n"
+					var/bleedtext = ""
+					if(H.bleed_rate <= 5)
+						bleedtext = "Negligible"
+					else if(H.bleed_rate <= 10)
+						bleedtext = "Minor"
+					else if(H.bleed_rate <= 15)
+						bleedtext = "Moderate"
+					else if(H.bleed_rate <= 20)
+						bleedtext = "Serious"
+					else if(H.bleed_rate <= 30)
+						bleedtext = "Critical!"
+					else if(H.bleed_rate > 30)
+						bleedtext = "Extreme!"
+					var/bandagetext = ""
+					if(H.bleedsuppress)
+						bandagetext = "Bandaged"
```

Comment:
as of right now I don't really see how to implement this proper without going through all limbs
```suggestion
```